packages/test/test-utils/src/loaderContainerTracker.ts

```diff
@@ -195,7 +195,10 @@ export class LoaderContainerTracker implements IOpProcessingController {
                 // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
                 debugWait(`Waiting container to be saved ${dirtyContainers.map((c) => this.containers.get(c)!.index)}`);
                 waitingSequenceNumberSynchronized = false;
-                await Promise.all(dirtyContainers.map(async (c) => new Promise((resolve) => c.once("saved", resolve))));
+                await Promise.all(dirtyContainers.map(async (c) => Promise.race(
+                    [new Promise((resolve) => c.once("saved", resolve)),
+                    new Promise((resolve) => c.once("closed", resolve))],
+                )));
```

Comment:
I think this is fine for now. The major annoyance was this function hanging when the container was closed with an error and the test would timeout. In this case, now it will no longer hang but if there's an error, the 'unexpected error in logs' mechanism would catch that..